src/patternfly/components/Pagination/pagination.scss

```diff
@@ -0,0 +1,90 @@
+@import "../../patternfly-imports";
+
+.pf-c-pagination {
+  // Base
+  --pf-c-pagination--FontSize: var(--pf-global--FontSize--sm);
+  --pf-c-pagination--child--MarginRight: var(--pf-global--spacer--lg);
+
+  // Dropdown
+  --pf-c-pagination__menu-text--Color: var(--pf-global--Color--200);
+
+  // Nav
+  --pf-c-pagination__nav--c-button--PaddingLeft: var(--pf-global--spacer--sm);
+  --pf-c-pagination__nav--c-button--PaddingRight: var(--pf-global--spacer--sm);
+  --pf-c-pagination__nav--c-button--FontSize: var(--pf-global--FontSize--md);
+
+  // Nav page select
+  --pf-c-pagination__nav-page-select--PaddingLeft: var(--pf-global--spacer--md);
+  --pf-c-pagination__nav-page-select--PaddingRight: var(--pf-global--spacer--md);
+  --pf-c-pagination__nav-page-select--child--MarginRight: var(--pf-global--spacer--xs);
+  --pf-c-pagination__nav-page-select--c-form-control--MaxWidth: #{pf-size-prem(48px)}; // custom value to show 2 numbers
+
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+  justify-content: flex-end;
+  font-size: var(--pf-c-pagination--FontSize);
+
+  * {
+    font-size: inherit;
+    white-space: nowrap;
+  }
+
+  > *:not(:last-child):not(.pf-c-pagination__current) {
+    margin-right: var(--pf-c-pagination--child--MarginRight);
+  }
+
+  // Top pagination
+  &:not(.pf-m-footer) {
+    justify-content: flex-end;
+
+    .pf-c-options-menu,
+    .pf-c-pagination__nav {
+      display: none;
+
+      @media screen and (min-width: $pf-global--breakpoint--md) {
+        display: flex;
+      }
+    }
+  }
+}
+
+// Nav
+.pf-c-pagination__nav {
+  display: inline-flex;
+  justify-content: flex-end;
+
+  .pf-c-button {
+    padding-right: var(--pf-c-pagination__nav--c-button--PaddingRight);
+    padding-left: var(--pf-c-pagination__nav--c-button--PaddingLeft);
+    font-size: var(--pf-c-pagination__nav--c-button--FontSize);
+  }
+}
+
+// Nav page element
+.pf-c-pagination__nav-page-select {
+  display: flex;
+  align-items: center;
+  padding-right: var(--pf-c-pagination__nav-page-select--PaddingRight);
+  padding-left: var(--pf-c-pagination__nav-page-select--PaddingLeft);
+
+  > *:not(:last-child) {
+    margin-right: var(--pf-c-pagination__nav-page-select--child--MarginRight);
+  }
+
+  .pf-c-form-control {
+    width: var(--pf-c-pagination__nav-page-select--c-form-control--MaxWidth);
```

Comment:
should this be `max-width`

Reply:
nice, looks like it needs to be width. updated the var